src/devices/cpu/sparc/sparcdasm.cpp

```diff
@@ -603,20 +603,13 @@ template <typename T> inline void sparc_disassembler::add_vis_op_desc(const T &d
 		m_vis_op_desc.insert(it);
 }
 
-inline void sparc_disassembler::pad_op_field(char *buf, char *&output) const
+inline void sparc_disassembler::pad_op_field(std::ostream &stream, std::streampos start_position) const
 {
-	while ((output - buf) < m_op_field_width) *output++ = ' ';
+	std::streamoff difference = stream.tellp() - start_position;
```

Comment:
Please make this `const`, and use constructor syntax like every other local in the file.